### PR TITLE
[ch80] Update readme

### PR DESCRIPTION
**Pull request by kimvcp**
https://app.clubhouse.io/cobda/story/80/update-readme-md

## Why?

This PR is for updating the content in the `README.md` file.

## Changes

Added necessary topics.

## Screenshot(s)

**Before** 

Screen Shot 2564-01-25 at 17 44 34

**After**

Screen Shot 2564-01-25 at 17 44 48

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
-# cobda-web
\ No newline at end of file
+**Cobda** is a community-like e-commerce website for selling and buying streetwear stuff. Plus, it's accessible anywhere and anytime through the internet.
```
Comment (LilBank) [inline README.md]:
```suggestion
**Cobda** is a community-like e-commerce website for selling and buying streetwears. Plus, it's accessible through any online devices.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+
+## Prerequisite
```
Comment (LilBank) [inline README.md]:
```suggestion
## Prerequisites
```

File: README.md [hunk 3]
```diff
@@ -1 +1,34 @@
...
+
+You must download the following to be able to run.
+
+- `Node.js (6.x or higher)` [download site](https://nodejs.org/en/download/package-manager/)
+- `local environment variable` **Please ask project owner privately for the environment variables.**
```
Comment (LilBank) [inline README.md]:
```suggestion
- `.env` **Please ask project owner privately for the environment variables.**
```

File: README.md [hunk 4]
```diff
@@ -1 +1,34 @@
-# cobda-web
\ No newline at end of file
+**Cobda** is a community-like e-commerce website for selling and buying streetwear stuff. Plus, it's accessible anywhere and anytime through the internet.
+
+## Prerequisite
+
+You must download the following to be able to run.
```
Comment (LilBank) [inline README.md]: I don't think this sentence is necessary this the title already has its own meaning.

**Review by LilBank — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1 +1,33 @@
-# cobda-web
\ No newline at end of file
+**Cobda** is a community-like e-commerce website for selling and buying streetwears. Plus, it's accessible through any online devices.
```
Comment (Raksani) [inline README.md]: Grammar check 👍🏻
```suggestion
**Cobda** is a streetwear e-commerce website where sellers and buyers meet each other with ease, driven by our desire to make community-like websites available to everyone in a friendly way. **Make certain** with a powerful AI function that automatically takes out inappropriate and irrelevant content to make sure your vibe goes as smoothly as possible. **Buy with confidence** by looking at product reviews and buyer's profile. This website runs perfectly on any online device. 
```

**Review by Raksani — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -1 +1,29 @@
-# cobda-web
\ No newline at end of file
+**Cobda** is a streetwear e-commerce website where sellers and buyers meet each other with ease, driven by our desire to make community-like websites available to everyone in a friendly way. **Make certain** with a powerful AI function that automatically takes out inappropriate and irrelevant content to make sure your vibe goes as smoothly as possible. **Buy with confidence** by looking at product reviews and buyer's profile. This website runs perfectly on any online device.
```
Comment (LilBank) [inline README.md]:
```suggestion
**Cobda** is a streetwear e-commerce website where sellers and buyers met with ease. With a community alike design, it's available for every genders and generations. **Make certain** with an autonomous feature that validate newly registered products at instant. Selling product is now faster than ever. **Buy with confidence** guaranteed by other customers. The review system builds up credits for the seller while giving confidence to the buyer vice versa. Finally, accessible through any devices with an internet connection 😎
```

Comment (kimvcp) [reply README.md]:
```suggestion
**Cobda** is a streetwear e-commerce website where sellers and buyers met with ease. With a community alike design, it's available for every gender and generation. **Make certain** with an autonomous feature that validates newly registered products instantly. Selling product is now faster than ever. **Buy with confidence** guaranteed by other customers. The review system builds up credits for the seller while giving confidence to the buyer vice versa. Moreover, accessible through any devices with an internet connection.
```

**Review by Kittinske15 — APPROVED** [review]

**Review by LilBank — APPROVED** [review]

**Pull request merged**